--- vite.config.ts
+++ vite.config.ts
@@ -14,10 +14,6 @@
 export default defineConfig({
   server: {
     port: 60000,
-    fs: {
-      // NOTE currently needed for embedding the `LiveStore` monorepo in another monorepo (e.g. under `/other-monorepo/submodules/livestore`)
-      allow: process.env.MONOREPO_ROOT ? [process.env.MONOREPO_ROOT] : [process.env.WORKSPACE_ROOT!],
-    },
   },
   worker: isProdBuild ? { format: 'es' } : undefined,
   optimizeDeps: {
@@ -29,13 +25,6 @@
       '@': path.resolve(__dirname, 'src'),
     },
   },
-  // NOTE This is only in here for convenience while developing the LiveStore devtools (feel free to remove it in your app)
-  // resolve: {
-  //   alias: {
-  //     '@livestore/devtools-react': path.resolve('../../../../packages/@livestore/devtools-react/src'),
-  //     // '@livestore/devtools-react': path.resolve('../../../../packages/@livestore/devtools-react/tmp-build/dist'),
-  //   },
-  // },
   plugins: [
     react(),
     livestoreDevtoolsPlugin({ schemaPath: './src/livestore/schema/index.ts' }),
