diff --git a/dist/cjs/internal/encoding/common.js b/dist/cjs/internal/encoding/common.js
index ebedb01aba08ed7435f603fcb3f72fbf7f08a60d..9aa77d64f29bd883bc6e1abe72a466f444591c51 100644
--- a/dist/cjs/internal/encoding/common.js
+++ b/dist/cjs/internal/encoding/common.js
@@ -1,18 +1,27 @@
 "use strict";
 
+globalThis.TextEncoder = require("text-encoding").TextEncoder;
+globalThis.TextDecoder = require("text-encoding").TextDecoder;
+
 Object.defineProperty(exports, "__esModule", {
-  value: true
+  value: true,
 });
-exports.isDecodeException = exports.encoder = exports.decoder = exports.DecodeExceptionTypeId = exports.DecodeException = void 0;
-var _Predicate = /*#__PURE__*/require("../../Predicate.js");
+exports.isDecodeException =
+  exports.encoder =
+  exports.decoder =
+  exports.DecodeExceptionTypeId =
+  exports.DecodeException =
+    void 0;
+var _Predicate = /*#__PURE__*/ require("../../Predicate.js");
 /** @internal */
-const DecodeExceptionTypeId = exports.DecodeExceptionTypeId = /*#__PURE__*/Symbol.for("effect/Encoding/errors/Decode");
+const DecodeExceptionTypeId = (exports.DecodeExceptionTypeId =
+  /*#__PURE__*/ Symbol.for("effect/Encoding/errors/Decode"));
 /** @internal */
 const DecodeException = (input, message) => {
   const out = {
     _tag: "DecodeException",
     [DecodeExceptionTypeId]: DecodeExceptionTypeId,
-    input
+    input,
   };
   if ((0, _Predicate.isString)(message)) {
     out.message = message;
@@ -21,10 +30,11 @@ const DecodeException = (input, message) => {
 };
 /** @internal */
 exports.DecodeException = DecodeException;
-const isDecodeException = u => (0, _Predicate.hasProperty)(u, DecodeExceptionTypeId);
+const isDecodeException = (u) =>
+  (0, _Predicate.hasProperty)(u, DecodeExceptionTypeId);
 /** @interal */
 exports.isDecodeException = isDecodeException;
-const encoder = exports.encoder = /*#__PURE__*/new TextEncoder();
+const encoder = (exports.encoder = /*#__PURE__*/ new TextEncoder());
 /** @interal */
-const decoder = exports.decoder = /*#__PURE__*/new TextDecoder();
+const decoder = (exports.decoder = /*#__PURE__*/ new TextDecoder());
 //# sourceMappingURL=common.js.map
