diff --git a/build/autolinking/findModules.js b/build/autolinking/findModules.js
index 9ccdb9e6f150c0f8bfe22b56be437f76407f23e9..8a118523317f4afa8fee4f3f099108d2d8b0affe 100644
--- a/build/autolinking/findModules.js
+++ b/build/autolinking/findModules.js
@@ -67,6 +67,9 @@ async function findModulesAsync(providedOptions) {
         }
     }
     const searchResults = Object.fromEntries(results.entries());
+
+    return searchResults
+
     // It doesn't make much sense to strip modules if there is only one search path.
     // (excluding custom native modules path)
     // Workspace root usually doesn't specify all its dependencies (see Expo Go),
