diff --git a/index.js b/index.js
index 9fef0037e087e7948214f7823daaa5e4612b2dac..adca0f290f098c59b63e37e45604cd2c9683f3be 100644
--- a/index.js
+++ b/index.js
@@ -1,8 +1,13 @@
+let routesInjected = false;
+
 export function starlightMarkdownIntegration(options) {
   return {
     name: "starlight-markdown",
     hooks: {
       "astro:config:setup": async ({ injectRoute }) => {
+        if (routesInjected) {
+          return;
+        }
         injectRoute({
           pattern: "/index.md",
           entrypoint: "starlight-markdown/markdown.md.js",
@@ -11,6 +16,7 @@ export function starlightMarkdownIntegration(options) {
           pattern: "/[...path]/index.md",
           entrypoint: "starlight-markdown/markdown.md.js",
         });
+        routesInjected = true;
       },
     },
   };
