--- examples/todomvc/vite.config.ts	2024-09-02 19:18:31.710742475 +0200
+++ examples-monorepo/examples/todomvc/vite.config.ts	2024-09-02 19:19:16.645308510 +0200
@@ -24,10 +24,19 @@
   server: {
     port: process.env.PORT ? Number(process.env.PORT) : 60_001,
     headers: credentiallessHeaders,
+    fs: {
+      // NOTE currently needed for embedding the `LiveStore` monorepo in another monorepo (e.g. under `/other-monorepo/submodules/livestore`)
+      // Feel free to remove this if you're just copying this example
+      allow: ['../../../..'],
+    },
   },
   preview: {
     headers: credentiallessHeaders,
   },
+  build: {
+    //   sourcemap: true,
+    //   minify: false,
+  },
   worker: isProdBuild ? { format: 'es' } : undefined,
   optimizeDeps: {
     // TODO remove once fixed https://github.com/vitejs/vite/issues/8427
